app/javascript/components/remove-catalog-item-modal.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Modal } from 'patternfly-react';
+
+const parseApiError = (error) => {
```

Comment:
This function could be extracted as a helper, because it will be surely used in other components.

I have this function as a helper in one of my PRs: https://github.com/ManageIQ/manageiq-ui-classic/pull/5297/files#diff-97a21497bae092187e30ce47ebd63b77

Reply:
@rvsia OK if I do that in a different PR?

Reply:
No problem! 👍